server/webapp/WEB-INF/rails.new/webpack/views/shared/new_modal.js.msx

```diff
@@ -55,11 +57,27 @@ const Modal = function (options) {
           destroy();
         }
       };
+
+      this.handleEnterKey = (e) => {
+        if (e.keyCode === ENTER_KEY_CODE) {
```

Comment:
this has been deprecated https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode